vocabularies/AI.json

```diff
@@ -0,0 +1,117 @@
+      { "rel": "latest-version", "href": "https://sap.github.io/odata-vocabularies/vocabularies/AI.json" },
+      { "rel": "describedby", "href": "https://github.com/sap/odata-vocabularies/blob/main/vocabularies/AI.md" }
+    ],
+    "Recommendations": [
+      {
+        "$Kind": "Function",
+        "$IsBound": true,
+        "@Common.Experimental": true,
+        "@Core.IsSignature": true,
+        "@Core.Description": "Computes AI-based recommendations for the bound entity and its related entities",
+        "@Core.LongDescription": "The function SHOULD compute the recommendations asynchronously, see [this diagram](../docs/recommendations.md).",
+        "$Parameter": [
+          { "$Name": "Entity", "$Type": "Edm.EntityType" },
+          {
+            "$Name": "AIRecommendationTargets",
+            "$Type": "AI.RecommendationTargetsType",
+            "@Core.Description": "Entities, properties or action/function parameters for which recommendations shall be retrieved",
+            "@Core.LongDescription": "Implementations of this function MAY omit this parameter.\n            If this parameter is omitted (by the implementation or in the invocation), all recommendations are retrieved.",
+            "@Core.OptionalParameter": {}
+          }
+        ],
+        "$ReturnType": {
+          "$Collection": true,
+          "$Type": "AI.PropertyRecommendationType",
+          "@Core.Description": "A collection of recommendations that differ in their `AIRecommendedFieldPath` property"
+        }
+      }
+    ],
+    "RecommendationTargetsType": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "URLs": {
+        "$Collection": true,
+        "@Core.IsURL": true,
+        "@Core.Description": "URLs (relative to the request URL) that address entities, properties or action/function parameters for which recommendations shall be retrieved",
+        "@Core.LongDescription": "The request `SalesOrder('A')/AI.Recommendations?AIRecommendationTargets={\"URLs\":[\"\",\"Items(10)\"]}`\n            retrieves recommendations for the entities `SalesOrder('A')` and `SalesOrder('A')/Items(10)`."
+      }
+    },
+    "PropertyRecommendationType": {
+      "$Kind": "ComplexType",
+      "@Common.Experimental": true,
+      "AIRecommendedFieldPath": {
+        "@Core.IsURL": true,
+        "@Core.Description": "URL (relative to the URL of the [`Recommendations`](#Recommendations) function request) addressing a property [OData-URL, section 4.6](https://docs.oasis-open.org/odata/odata/v4.01/os/part2-url-conventions/odata-v4.01-os-part2-url-conventions.html#sec_AddressingaProperty) that is targeted by this recommendation",
+        "@Core.LongDescription": "The request `SalesOrder('A')/AI.Recommendations?AIRecommendationTargets=...`\n            might retrieve a recommendation with `\"AIRecommendedFieldPath\": \"Items(10)/Product\"`."
+      },
```

Comment:
For consistency reason, can we keep this comment also to indicate that the backend needs to send the full path back to the client and not relative